lxd/container_post.go

```diff
@@ -171,10 +166,15 @@ func containerPost(d *Daemon, r *http.Request) response.Response {
 		stateful = req.Live
 	}
 
+	inst, err := instance.LoadByProjectAndName(d.State(), project, name)
+	if err != nil {
+		return response.SmartError(err)
+	}
+
 	if req.Migration {
 		if targetNode != "" {
 			// Check whether the container is running.
-			if inst != nil && inst.IsRunning() {
+			if inst.IsRunning() {
```

Comment:
@monstermunchkin the `IsRunning! function is going to try and talk to liblxc, so you can only do that if the sourceNodeOffline so I think we'll need to make allowances for that here.

Reply:
Thanks. Pushed a commit which addresses this.